bp-web-client/angular-app/src/app/services/scroll-to.service.ts

```diff
@@ -1,13 +1,20 @@
-import { Injectable } from '@angular/core';
-
+import { Injectable } from "@angular/core";
+import { BehaviorSubject } from "rxjs";
 
 @Injectable({
-  providedIn: 'root'
+  providedIn: "root",
```

Comment:
Use single quotes instead of double quotes.

Reply:
changed to single quotes